bedrock/firefox/templates/firefox/mobile.html

```diff
@@ -26,9 +26,20 @@
     <header class="main-header">
       <div class="content">
         <div class="header-content">
-          <h1>{{ _('Firefox for mobile') }}</h1>
-          <h3>{{ _('The speed you need with the privacy you want on all your devices.') }}</h3>
-          <p>{{ _('Choose one or download them all for iOS and Android.') }}</p>
+          <div class="header-title">
+              <h1>{{ _('Firefox for mobile') }}</h1>
+              {% if l10n_has_tag('fast-012018') %}
```

Comment:
It doesn't break anything, but please use underscore for tags

P.S. indentation seems off for the endif closing tag